plugins/dominant-color-images/tests/test-dominant-color.php

```diff
@@ -42,7 +42,7 @@ public function test_dominant_color_metadata( string $image_path, array $expecte
 	 *
 	 * @dataProvider provider_get_dominant_color
 	 *
-	 * @covers ::dominant_color_get_dominant_color
+	 * @covers helper::dominant_color_get_dominant_color
```

Comment:
This doesn't seem right? The `::` here refers to the global namespace.

```suggestion
	 * @covers ::dominant_color_get_dominant_color
```

Reply:
`: :` refers to the global namespace, and it works for rest of the functions however, when I'm trying to generate coverage report, for the functions where I've added `helper` & `hooks` it's showing 0 coverage without file reference.

### Previous Result : 

- When setting annotation to `* @covers ::dominant_color_get_dominant_color`  & `* @covers ::dominant_color_img_tag_add_dominant_color`


- `helper.php`
<img width="1437" alt="image" src="https://github.com/user-attachments/assets/0c0250e4-b27e-4e2e-aa18-9d1ee6e88f96" />

- `hooks.php`
<img width="1431" alt="image" src="https://github.com/user-attachments/assets/a0c70ab4-441f-41dd-9f61-ff9fe17a3ac7" />

### New Result : 

- When setting annotation to `* @covers helper::dominant_color_get_dominant_color`  & `* @covers hooks::dominant_color_img_tag_add_dominant_color`

- `helper.php`
<img width="1450" alt="image" src="https://github.com/user-attachments/assets/e3efde93-4260-4872-8904-cae8bb862f53" />


- `hooks.php`
<img width="1444" alt="image" src="https://github.com/user-attachments/assets/a6e6761a-93ee-4daa-93ba-7a872a5b0059" />


So how should I proceed with this?

cc : @westonruter

Reply:
Strange. I don't know why that would be. I also don't understand adding `hooks` and `helper` to the beginning of those functions. They would seem to indicate there are classes called `hooks` and `helper`, which there aren't.

You're seeing this with local coverage report, but are you also seeing it with the Codecov coverage report?

Reply:
> They would seem to indicate there are classes called hooks and helper, which there aren't.

Exactly, still for some reason it seems to work.


> You're seeing this with local coverage report, but are you also seeing it with the Codecov coverage report?

Yes, in Codecov coverage report also the same is happening.

@westonruter

Reply:
@thelovekesh any ideas?

Reply:
@sarthak-19 can you please upload the xml coverage report? maybe that can help analyze how code coverage driver is looking for things?

Reply:
@thelovekesh  Here's the link to xml report : https://drive.google.com/drive/folders/1Mj8KLo6IcnqsYstaZkRekREtkBzn0-9z?usp=sharing